code/modules/mob/living/carbon/human/human_stripping.dm

```diff
@@ -182,6 +182,17 @@ GLOBAL_LIST_INIT(strippable_human_layout, list(
 	user.log_message(log_message, LOG_ATTACK, color="red", log_globally=FALSE)
 	item.add_fingerprint(src)
 
+	if(iscarbon(user))
+		var/mob/living/carbon/carbon_user = user
+
+		if(istype(item, /obj/item/reagent_containers/syringe/)  && !istype(carbon_user.gloves, /obj/item/clothing/gloves/tackler))
```

Comment:
maybe instead of using istype and specifying exact gloves add some flag, or make it based on armor values or smth